docs/source/deployment/now.md

```diff
@@ -62,15 +62,15 @@ The `now` command deploys right away and attempts to start the server. This spec
 
 [Now](https://zeit.co/now) supports automatic deployment from GitHub on pull requests.
 
-To set up automatic deployment, visit [https://zeit.co/github](https://zeit.co/github) and click on the "Setup Now" button.
+To set up automatic deployment, visit [https://zeit.co/github](https://zeit.co/github) and click **Setup Now**.
 
 After signing in with GitHub, the [Now](https://zeit.co/now) GitHub app can be added to any account or organization. Once installed, it's possible to choose which repositories that [Now](https://zeit.co/now) can run on, allowing new deployments on Pull Requests.
 
 ![now permissions setup](../images/deployment/zeit/now-github-permissions.png)
 
 ## Setting environment variables
 
-The `graphql-server-example` project requires environment variables to enable reporting to Apollo Engine. To deploy to Now with environment variables, the `-e` flag can be used followed by the variables like so:
+The `graphql-server-example` project requires environment variables to enable reporting to Apollo Graph Manager. To deploy to Now with environment variables, the `-e` flag can be used followed by the variables like so:
```

Comment:
Maybe do this.

```suggestion
The `graphql-server-example` project requires environment variables to enable reporting to Graph Manager. To deploy to Now with environment variables, the `-e` flag can be used followed by the variables like so:
```

Reply:
This is an exceptional case where this is the only reference to Graph Manager in the article, so it should probably name it fully